tests/coalaDeleteOrigTest.py

```diff
@@ -49,3 +50,12 @@ def test_normal_running(self):
             retval = coala_delete_orig.main(section=section)
             self.assertEqual(retval, 0)
             self.assertFalse(os.path.isfile(temporary[1]))
+
+    def test_log_printer_warning(self):
+        with retrieve_stderr() as stderr:
+            configure_logging()
+            coala_delete_orig.main(log_printer='SomeLogPrinter',
+                                   section=self.section)
+            output = stderr.getvalue()
+            self.assertIn('[WARNING]', output)
+            self.assertIn('Using log_printer is deprecated', output)
```

Comment:
I didnt use a regex in this commit as `color=False` was not available.
Added the regex in a later commit.